### DISCUSSION TITLE
Clearing a line plot?

### DISCUSSION THREAD

**Discussion by stur86** [zauberzeug/nicegui#1316] [Ideas / Feature Requests]
### Question

Line plots are a really useful element, but I can't find an obvious way to clear them back to a blank slate, which sometimes may be needed. I tried even accessing the underlying `fig` element but it didn't work. Am I missing something or do I have to use a regular PyPlot element?

**Comment by falkoschindler** [comment]
Hi @stur86!
You're right, there is no easy way to clear a line plot. I just came up with the following recipe:
```py
line_plot.x.clear()
for y in line_plot.Y:
    y.clear()
for line in line_plot.lines:
    line.set_data([], [])
line_plot._convert_to_html()
line_plot.update()
```
But we should provide a proper clear method. I'll convert this discussion into a feature request. 🙂

**Reply by falkoschindler** [reply]
See PR https://github.com/zauberzeug/nicegui/pull/1317.

**Reply by stur86** [reply]
Great, thanks! I expect this will then be included in the next release? For now I managed by setting `x` and `Y` manually to empty arrays.

**Reply by falkoschindler** [reply]
Yes, this will be released shortly with version 1.3.7.
